specification/cognitiveservices/data-plane/Face/v1.0/Face.json

```diff
@@ -1550,12 +1194,7 @@
         ],
```

Comment:
Is this operation consuming json / streams? It doesnt look like it

Reply:
Fixed